apps/challenges/models.py

```diff
@@ -220,3 +220,15 @@ class ChallengeConfiguration(TimeStampedModel):
     class Meta:
         app_label = 'challenges'
         db_table = 'challenge_zip_configuration'
+
+
+class StarChallenge(TimeStampedModel):
```

Comment:
I think one analytics can be to track an unstar action after a user has starred a challenge. This may be too much thinking, but this is something which the challenge host should know